plugins/woocommerce/src/Admin/Features/MarketingRecommendations/DefaultMarketingRecommendations.php

```diff
@@ -107,6 +107,24 @@ public static function get_all() {
 				),
 				'tags'           => array(),
 			),
+			array(
+				'title'          => 'Facebook for WooCommerce',
+				'description'    => __( 'List products and create ads on Facebook and Instagram.', 'woocommerce' ),
+				'url'            => "https://woocommerce.com/products/facebook/{$utm_string}",
+				'direct_install' => true,
+				'icon'           => "{$icon_dir_url}/facebook.svg",
+				'product'        => 'facebook-for-woocommerce',
+				'plugin'         => 'facebook-for-woocommerce/facebook-for-woocommerce.php',
+				'categories'     => array(
+					$marketing,
+				),
+				'subcategories'  => array(
+					$sales_channels,
+				),
+				'tags'           => array(
+					$built_by_woocommerce,
+				),
```

Comment:
I believe we're leaving off the "Built by WooCommerce badge" pcTzPI-Un-p2#comment-1299
```suggestion
				'tags'           => array(),
```